scripts/zones/Aht_Urhgan_Whitegate/npcs/Survival_Guide.lua

```diff
@@ -0,0 +1,12 @@
+require("scripts/globals/survival_guide")
+
+local eventId = 8500
+local guideIndex = 5
```

Comment:
The guide index is the the actual index (option value) that get's passed around to the event handlers so that's what I'm indexing off of. I do have a separate map where it's zone id to index so I could use that so there is less to pass in.